code/datums/traits/_quirk.dm

```diff
@@ -10,6 +10,7 @@
 	var/medical_record_text //This text will appear on medical records for the trait. Not yet implemented
 	var/mood_quirk = FALSE //if true, this quirk affects mood and is unavailable if moodlets are disabled
 	var/mob_trait //if applicable, apply and remove this mob trait
+	var/process = FALSE // Does this proc use on_process()?
```

Comment:
Cleaner than just putting START_PROCESSING in the New() of every quirk that uses on_process due to the quantity of them. For now at least